docs/reference/monitoring/how-monitoring-works.asciidoc

```diff
@@ -5,28 +5,71 @@
 <titleabbrev>How it works</titleabbrev>
 ++++
 
-Each {es} node, {ls} node, {kib} instance, and Beat instance is considered
-unique in the cluster based on its persistent UUID, which is written to the
-<<path-settings,`path.data`>> directory when the node or instance starts.
+Each monitored {stack} component is considered unique in the cluster based on
+its persistent UUID, which is written to the <<path-settings,`path.data`>>
+directory when the node or instance starts.
+
+****
+**Question for reviewers:**
+
+Is the following paragraph correct? Do all of the monitored products listed in
+the overview have UUIDs?
+****
 
 Monitoring documents are just ordinary JSON documents built by monitoring each
 {stack} component at a specified collection interval. If you want to alter the
 templates for these indices, see <<config-monitoring-indices>>.
 
-{metricbeat} is used to collect monitoring data and to ship it directly to the
-monitoring cluster.
+****
+**Question for reviewers:**
+
+What do Elastic Agent users need to know about configuring indices? I think the
+topic about configuring indices only applies to legacy collectors, but let me
+know if I'm incorrect. If it's only relevant for legacy collectors, I will
+remove the link above.
```

Comment:
Thanks for clarifying. I'm going to put this info in the topic about configuring indices because it seems out of place in the topic that's meant to describe how monitoring works. (TBH I don't think the topic really achieves its stated goal, but we'll ignore that for now.) I'll push the commit in a bit so you can see what I've done.